docs/developer/architecture/alerting.rst

```diff
@@ -43,3 +43,263 @@ maintenance operation.
 .. note:: Alertmanager configuration documentation is available here_
 
 .. _here: https://prometheus.io/docs/alerting/configuration/
+
+Design Choices
+--------------
+
+To be able to edit existing rules, add new ones, etc., and in ordrer to keep
+these changes across restorations, upgrades and downgrades, we need to put in
+place some mechanisms to configure Prometheus and Alertmanager and persist
+these configurations.
+
+For the persistence part, we will rely on what has been done for
+:doc:`CSC<configurations>` (Cluster and Services Configurations), and use
+the already defined resources for :term:`Alertmanager` and :term:`Prometheus`.
+
+Custom Prometheus Alert Rules
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We will use the already existing ``metalk8s-prometheus-config``
+:term:`ConfigMap` to store the :term:`Alertmanager` configuration
+customizations.
```

Comment:
We are using the existing `metalk8s-prometheus-config` ConfigMap to store `alerting rules` which is part of Prometheus and not Alertmanager configuration customizations? no??.

I think this should read 
```We will use the already existing metalk8s-prometheus-config ConfigMap to store custom alerting rules```

Reply:
Yes, bad copy/paste :D